src/prometheus_alert_rules/flapping.rules

```diff
@@ -0,0 +1,14 @@
+groups:
+- name: AlertRules
+  rules:
+  - alert: AlertRulesFlapping
+    expr: changes(ALERTS_FOR_STATE{severity = "warning"}[1h]) > 5
+    for: 5m
+    labels:
+      severity: warning
+    annotations:
+      summary: Host '{{labels.instance}}' suffers from flapping alerts.
```

Comment:
Would this alert contribute to alert fatigue?
I.e. if we already have a flapping alert, what's the point in adding another alert telling us that it's flapping?

Reply:
Flapping does not necessarily mean that alertmanager is routing it through, as it could be changing state to and from `pending`, meaning the condition is triggered, but not for long enough. This would catch that.

Reply:
I would agree with @sed-i here in that we should think harder about this. This is necessarily only going to increase (or keep the same) the amount of pages we get, and we are _already_ in a huge trouble regarding that amount.

If we want to make a comparison with LMA, Nagios would deliberately stop sending notifications for flapping checks. We're effectively doing the exact opposite with this. I would take a step back and think why are we doing this. @err404r, maybe we should have a conversation regarding what led you to initially file #145. If possible, we should try to address that particular situation if applicable in a more specific way.

Reply:
About alerts fatigue here, the idea is actually to reduce it by having this alert. Alert Manager has inhibition rules. So we could use this alert to prevent flapping alert from firing. So as long as rule active there will be only one constantly active alert. Instead of flapping alert

Reply:
I'm with @err404r. You would set up an inhibition rule saying that if there's already an alert on this one, don't fire another one, which would mean one flapping alert instead of 10 individual ones for each flap.

Reply:
If we inhibit the first alert, would it still be counted in the "flapping" expression when inhibited?
If it's inhibited "early" then it would cause the flapping alert to stop firing.